core/domain/exp_services_test.py

```diff
@@ -3874,6 +3876,24 @@ def test_update_exploration_with_invalid_commit_message(self):
                     'new_value': 'new title'
                 })], feconf.COMMIT_MESSAGE_ACCEPTED_SUGGESTION_PREFIX)
 
+    def test_update_exploration_does_nothing_if_create_stats_model_fails(self):
+        swap_create_stats_model = (
+            self.swap(stats_services, 'create_stats_model', lambda: 1 / 0))
+        assert_raises = self.assertRaises(Exception)
```

Comment:
Always prefer assertRaisesRegexp.

Also maybe name this something like swap_assert_raises_regexp for parity with the above.

Reply:
Sorry, you're right about the swap prefix. Thanks for the catch!